src/omnisharp/server.ts

```diff
@@ -319,7 +319,8 @@ export abstract class OmnisharpServer {
             cleanupPromise = new Promise<void>((resolve, reject) => {
                 const killer = exec(`taskkill /F /T /PID ${this._serverProcess.pid}`, (err, stdout, stderr) => {
                     if (err) {
-                        return reject(err);
+                        reject(err);
+                        return;
```

Comment:
Any reason you're changing these? It's perfectly legal to return a void-returning function.